src/shipit_signoff/shipit_signoff/swagger/swagger.yaml

```diff
@@ -0,0 +1,190 @@
+---
```

Comment:
This is autogenerated.  api.yml is what I've been changing, and swagger puts this in place to actually serve the api